builtin/providers/aws/structure.go

```diff
@@ -1567,3 +1567,24 @@ func flattenPolicyAttributes(list []*elb.PolicyAttributeDescription) []interface
 
 	return attributes
 }
+
+// Takes a value containing JSON string and passes it through
+// the JSON parser to normalize it, returns either a parsing
+// error or normalized JSON string.
+func normalizeJsonString(jsonString interface{}) (string, error) {
+	var j interface{}
+
+	if jsonString == nil || jsonString.(string) == "" {
+		return "", nil
+	}
+
+	s := jsonString.(string)
+
+	err := json.Unmarshal([]byte(s), &j)
+	if err != nil {
+		return s, err
+	}
+
+	bytes, _ := json.Marshal(j)
```

Comment:
I'm assuming that nothing can go wrong in marshalling an object which we were previously able to successfully unmarshal... so I guess 👍  

I am aware this is more or less copy-paste of the existing function and it's rather a rhetorical question.